test/e2e/k8s_util.go

```diff
@@ -81,25 +82,24 @@ func (k *KubernetesUtils) GetPods(ns string, key string, val string) ([]v1.Pod,
 
 // Probe execs into a Pod and checks its connectivity to another Pod.  Of course it assumes
 // that the target Pod is serving on the input port, and also that ncat is installed.
-func (k *KubernetesUtils) Probe(ns1, pod1, ns2, pod2 string, port int32) (bool, error) {
-	fromPods, err := k.GetPods(ns1, "pod", pod1)
+func (k *KubernetesUtils) Probe(ns1, pod1, ns2, pod2 string, port int32, usePodName bool) (bool, error) {
+	fromPods, err := k.GetPodsByLabel(ns1, "pod", pod1)
 	if err != nil {
-		return false, errors.WithMessagef(err, "unable to get pods from ns %s", ns1)
+		return false, fmt.Errorf("unable to get pods from ns %s: %v", ns1, err)
 	}
 	if len(fromPods) == 0 {
-		return false, errors.New(fmt.Sprintf("no pod of name %s in namespace %s found", pod1, ns1))
+		return false, fmt.Errorf("no Pod of label pod=%s in namespace %s found", pod1, ns1)
 	}
 	fromPod := fromPods[0]
 
-	toPods, err := k.GetPods(ns2, "pod", pod2)
+	toPods, err := k.GetPodsByLabel(ns2, "pod", pod2)
 	if err != nil {
-		return false, errors.WithMessagef(err, "unable to get pods from ns %s", ns2)
+		return false, fmt.Errorf("unable to get pods from ns %s: %v", ns2, err)
 	}
 	if len(toPods) == 0 {
-		return false, errors.New(fmt.Sprintf("no pod of name %s in namespace %s found", pod2, ns2))
+		return false, fmt.Errorf("no Pod of label pod=%s in namespace %s found", pod2, ns2)
 	}
 	toPod := toPods[0]
```

Comment:
nit: not your code but Pod and Namespace should be capitalized

Reply:
Done